site/src/sections/Chapters/Accordion.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import openIcon from "../../assets/images/chapters/open.svg";
+import closeIcon from "../../assets/images/chapters/close.svg";
+
+const Accordion = ({ area, name, topics }) => {
+  const [open, setOpen] = useState(false);
+  return (
+    <>
+      <div className="accordion-title" onClick={() => setOpen(prevState => !prevState)}>
+        <p>{area}</p>
+        <p>{name}</p>
+        <button onClick={() => setOpen(prevState => !prevState)}>
+          {open ? (<img src={openIcon} alt="open" />) : (<img src={closeIcon} alt="close" />) }
+        </button>
+      </div>
+      {open && (
+        <div className="accordion-content">
+          {topics.map((topic, idx) => (
+            <div className="topic" key={idx}>
+              <p>{topic.name}</p>
+              <p>{topic.type}</p>
+            </div>
+          )) }
+        </div>
+      )}
+    </>
+  );
+};
+
+export default Accordion;
```

Comment:
The `Accordion` can be moved to the component folder.
P.S: Thanks for making it generic :)

Reply:
No worries and done!